src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetCommandInformationHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices.Services;
+using MediatR;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.PowerShell.EditorServices.Handlers
+{
+    [Serial, Method("powerShell/getCommandInformation")]
+    public interface IGetCommandInformationHandler : IJsonRpcRequestHandler<GetCommandInformationParams, List<PSCommandInformationMessage>> { }
+
+    public class GetCommandInformationParams : IRequest<List<PSCommandInformationMessage>> {
+        public string Name { get; set; }
+    }
+
+    /// <summary>
+    /// Describes the message to get the details for a single PowerShell Command
+    /// from the current session
+    /// </summary>
+    public class PSCommandInformationMessage
+    {
+        public string Name { get; set; }
+        public string ModuleName { get; set; }
+        public string DefaultParameterSet { get; set; }
+        public Dictionary<string, ParameterMetadata> Parameters { get; set; }
+        public System.Collections.ObjectModel.ReadOnlyCollection<CommandParameterSetInfo> ParameterSets { get; set; }
+    }
+
+    public class GetCommandInformationHandler : IGetCommandInformationHandler
+    {
+        private readonly ILogger<GetCommandInformationHandler> _logger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_logger' or refactor the code to use its value.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4980809)